app/static/js/initialize_map.js

```diff
@@ -0,0 +1,155 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+//Initial map center coordinates
+INITIAL_CENTER_LAT = 39.952;
```

Comment:
Yes, I think so because if we are denied access to user location (the browser asks for permission from the user to use location), then we still need to center the map somewhere. Do you have other suggestions?